components/movementsensor/gpsrtk/gpsrtk.go

```diff
@@ -567,6 +567,14 @@ func (g *RTKMovementSensor) LinearVelocity(ctx context.Context, extra map[string
 	return g.nmeamovementsensor.LinearVelocity(ctx, extra)
 }
 
+// LinearAcceleration passthrough.
+func (g *RTKMovementSensor) LinearAcceleration(ctx context.Context, extra map[string]interface{}) (r3.Vector, error) {
+	if g.lastError != nil {
+		return r3.Vector{}, g.lastError
+	}
+	return g.nmeamovementsensor.LinearAcceleration(ctx, extra)
```

Comment:
just clarifying here, but are we planning in the future to properly implement LinearAcceleration on the nmea sensors? Asking because as of now this appears to defer to a struct that raises the unimplemented error

Reply:
I have no idea if it will ever be implemented, but I like the way this is currently written. If it ever gets implemented, it'll be in the nmea sensor, and then we'll get it here, too, "for free."